backend/src/main/kotlin/outcobra/server/exception/BadDtoException.kt

```diff
@@ -0,0 +1,5 @@
+package outcobra.server.exception
+
+class BadDtoException(override val message: String = "received invalid Dto") : BadRequestException(message) {
+    constructor(errorEnum: ErrorEnum) : this(errorEnum.i18n)
+}
```

Comment:
I think you should put `INVALID_DTO` as default value here and remove the message default string